fcrepo-http-api/src/main/resources/views/common.vsl

```diff
@@ -46,7 +46,7 @@
 #end
 
 #macro( isObject $sub )
-#set($type = $anynode.createURI("http://fedora.info/definitions/v4/repository#mixinTypes"))
+#set($type = $helpers.createURI("http://fedora.info/definitions/v4/repository#mixinTypes"))
```

Comment:
Although it does not appear that this macro is being used, I wanted to make sure that it at least worked.
https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/graph/Node_ANY.html no longer has the `createURI` method in Jena 3.